plugins/org.eclipse.elk.alg.layered/src/org/eclipse/elk/alg/layered/Layered.melk

```diff
@@ -909,13 +913,41 @@ group edgeLabels {
 /* ------------------------
  *    miscellaneous
  * ------------------------*/
-advanced option considerModelOrder: OrderingStrategy {
-    label "Consider model order"
-    description
-        "Preserves the order of nodes and edges in the model file if this does not lead to edge crossings or
-         conflicts between the ordering or edges and nodes."
-    default = OrderingStrategy.NONE
-    targets parents
+group considerModelOrder {
+    option strategy: OrderingStrategy {
+        label "Consider model order"
+        description
+            "Preserves the order of nodes and edges in the model file if this does not lead to edge crossings or
+             conflicts between the ordering or edges and nodes."
+        default = OrderingStrategy.NONE
+        targets parents
+    }
+    option dummyStrategy: DummyNodeStrategy {
+        label "Dummy node strategy"
+        description
+            "Indicates whether dummy nodes are sorted under, over, or equal to nodes that have no connection to a
```

Comment:
I will add you r suggestions to the description. Over and under does not change for a right to left layout since the order of the nodes in the different layers does not change if one switches from left-to-right to right-to-left.